doc/usermanual/darkroom/panels/histogram.xml

```diff
@@ -16,36 +16,244 @@
-      <colspec colwidth="6*"/>
-      <colspec colwidth="4*"/>
-      <tbody>
-        <row>
-          <entry>
-            <para>
-              This shows a histogram of the developed image's light levels. In its default state
-              curves for all three RGB color channels are displayed. You can toggle the colored
-              squares to enable or disable specific color channels. A curve button is also
-              provided to toggle between linear view, logarithmic view and waveform view.
-            </para>
-
-            <para>
-              The histogram is directly linked to the <emphasis>exposure</emphasis> module
-              described in <xref linkend="exposure"/>, and you can operate some of the exposure
-              module's controls from the histogram. You can left-click towards the right hand
-              side of the histogram and then drag right to increase or drag left to decrease the
-              exposure. In a similar manner you can control the black level by clicking and
-              dragging in the left hand side. Double-clicking in the histogram resets the
-              exposure module's parameters to their defaults.
-            </para>
-          </entry>
-          <entry>
-            <graphic fileref="darkroom/images/panel_histogram.png" scalefit="1" width="80%" align="center" />
-          </entry>
-        </row>
-      </tbody>
-    </tgroup>
-  </informaltable>
+  <indexterm>
+    <primary>waveform</primary>
+  </indexterm>
+
+  <indexterm>
+    <primary>RGB parade</primary>
+  </indexterm>
+
+  <indexterm>
+    <primary>scopes</primary>
+  </indexterm>
+
+  <sect3>
+
+    <title>Overview</title>
+
+    <informaltable frame="none">
+      <tgroup cols="2" colsep="0" rowsep="0">
+        <colspec colwidth="6*"/>
+        <colspec colwidth="4*"/>
+        <tbody>
+          <row>
+            <entry>
+	      <para>
+		This is a graphical depiction of aspects of the developed image. Moving the
+		mouse over the panel displays buttons to adjust the display. The leftmost button
+		allows for cycling the mode between a "regular" and a waveform-style
+		histogram. The next button over adjusts how the data for the current mode is
+		displayed. The three rightmost colored squares are toggles which enable or
+		disable the display of specific color channels.
+	      </para>
+
+	      <para>
+		When the mouse is over the histogram panel, scrolling while holding down the
+		<emphasis>Ctrl</emphasis> key will change the height of the panel. You can
+		hide/show the histogram entirely with a keyboard shortcut. The default shortcut
+		is <emphasis>Shift+Ctrl+H</emphasis>.
+	      </para>
+
+	      <para>
+		It is possible to assign keyboard shortcuts to cycle through or switch histogram
+		mode/type. Separate shortcuts can be set for the darkroom and tethering
+		views. By default, none of these shortcuts are associated with keys.
+	      </para>
+
+	      <para>
```

Comment:
This is really for the releases notes and should not be part of the documentation I would say.

Reply:
@TurboGit Were you thinking to remove the entire paragraph ("It is possible to assign...none of these shortcuts are associated.") or just the last sentence ("By default...with keys.")? I'm happy to remove either/both...

Reply:
Sorry a bit late, my idea was to remove only:

`The histogram is currently disabled in tethering view, pending a code fix.`

The other paragraph are ok to me.

Reply:
OK! I did this, I hope...